snoop/data/views.py

```diff
@@ -724,3 +725,10 @@ def processing_status(request, hash):
     if total_count != 0 and result['done_count'] == total_count:
         result['finished'] = True
     return JsonResponse(result)
+
+
+def get_nextcloud_collections(request):
```

Comment:
let's put this view in the common app, since it only accesses the common models.